tools/console/console.js

```diff
@@ -674,7 +647,7 @@ _.extend(Console.prototype, {
     }
 
     return width;
-  },
+  }
 
   // This can be called during long lived operations; it will keep the spinner spinning.
   // (This code used to be in Patience.nudge)
```

Comment:
Wow, `Patience.nudge` used to exist? No one ran the code through `oxymoron-lint`? 🙂